.buildkite/react-native-navigation-pipeline.full.yml

```diff
@@ -90,32 +92,34 @@ steps:
             - exit_status: "*"
               limit: 1
 
-      - label: ':building_construction: :mac: Build react-native-navigation test fixture - RN {{matrix.rn_version}} (New Arch)'
-        key: "build-react-native-navigation-ios-fixture-new-arch-full"
-        timeout_in_minutes: 15
-        agents:
-          queue: "macos-12-arm"
-        env:
-          BUILD_IOS: "true"
-          NODE_VERSION: "18"
-          REACT_NATIVE_NAVIGATION: "true"
-          RCT_NEW_ARCH_ENABLED: "true"
-          RN_VERSION: "{{matrix.rn_version}}"
-          DEVELOPER_DIR: "/Applications/Xcode14.app"
-          NOTIFIER_VERSION: '8.0.0'
-        artifact_paths:
-          - "test/react-native/features/fixtures/generated/react-native-navigation/**/reactnative.ipa"
-        commands:
-          - bundle install
-          - ./bin/generate-react-native-fixture
-        matrix:
-          setup:
-            rn_version:
-              - "0.71"
-        retry:
-          automatic:
-            - exit_status: "*"
-              limit: 1
+      # Skipped due to issues with react-native-navigation when new arch is enabled
```

Comment:
Up until now these tests were actually running against the old architecture because we were setting `RCT_NEW_ARCH_ENABLED` to `true` instead of `1`

Reply:
The environment variable is used by React Native to enable/disable new arch for iOS builds, and it doesn't accept `true`, which is why the script now enforces `1`/`0`